cpp/include/cugraph_c/community_algorithms.h

```diff
@@ -177,6 +177,41 @@ double cugraph_hierarchical_clustering_result_get_modularity(
  */
 void cugraph_hierarchical_clustering_result_free(cugraph_hierarchical_clustering_result_t* result);
 
+/**
+ * @brief     Compute ECG clustering
+ *
+ * @param [in]  handle        Handle for accessing resources
+ * @param [in/out] rng_state  State of the random number generator, updated with each call
+ * @param [in]  graph         Pointer to graph.  NOTE: Graph might be modified if the storage
+ *                            needs to be transposed
+ * @param [in]  min_weight    Minimum edge weight in final graph
+ * @param [in]  ensemble_size The number of Louvain iterations to run
+ * @param [in]  max_level     Maximum level in hierarchy for final Louvain
```

Comment:
`max_level`, `threshold` and `resolution` are new flags that needs to be exposed to the PLC and python API.

Reply:
It's already exposed to PLC, and will be exposed to python in next PR when I remove all legacy stuffs.